src/spryker/docker/image/console/root/lib/task/skeleton/apply.sh.twig

```diff
@@ -29,9 +29,8 @@ function merge_composer_json()
     run "jq -s '.[0] * .[1]' $core_file $override_file > $merged_file"
     run "mv $merged_file $core_file"
 
-    local new_composer_packages="$(jq -r '.["require-dev"] | keys[]' $override_file | tr '\n' ' ')"
+    local new_composer_packages=""
+    new_composer_packages="$(jq -r '.["require-dev"] * .["require"] | keys[]' $override_file | tr '\n' ' ')"
```

Comment:
why defining empty var first and then reassigning with different value?

Reply:
Shellcheck recommendation from https://github.com/koalaman/shellcheck/wiki/SC2155